src/contrib/monitoring/check_zookeeper.py

```diff
@@ -169,11 +169,16 @@ def __init__(self, host='localhost', port='2181', timeout=1):
     def get_stats(self):
         """ Get ZooKeeper server stats as a map """
         data = self._send_cmd('mntr')
+        stat = self._parse_stat(self._send_cmd('stat'))
         if data:
-            return self._parse(data)
+            mntr = self._parse(data)
+            missing = ['zk_zxid', 'zk_zxid_counter', 'zk_zxid_epoch']
```

Comment:
why not just include all of the output from stat?

Reply:
Because the rest of the stat output is already present in the mntr output. Previously for servers that support the mntr command the script wouldn't run stat at all: I only added it because the zxid info is not in mntr.

Reply:
How about changing the below to:

```python
for s in stat:
    if s not in mntr:
        mntr[s] = stat[s]
```

It will have the same effect now, but if future things are added to stat but not mntr we'd pick them up.

Reply:
@afine are you ok with this? Looks like a useful idea that I'd like to include once you're satisfied.